### PR TITLE
Feature/landing page

### PR DESCRIPTION
**Pull request by engshorouq**
relates #23 
create a landing page for the website

### CODE REVIEW

File: client/src/Component/Pages/Landing/style.css [hunk 1]
```diff
@@ -0,0 +1,48 @@
+.land {
+  position: relative;
+  min-height: 87vh;
+}
+
+.content {
+  background: white;
+  padding: 20px;
```
Comment (dubhcait) [inline client/src/Component/Pages/Landing/style.css]: I think it looks slightly better with padding:32px.

File: client/src/App.js [hunk 2]
```diff
@@ -1,11 +1,20 @@
...
+import {Land} from './Component/Pages';
+
 import "./App.css";
 
 
 function App() {
   return (
     <div className="App">
-   
+      <Router>
```
Comment (IsraaSulaiman) [inline client/src/App.js]: Can you remove app. js from your pull request?

Comment (engshorouq) [reply client/src/App.js]: ok

File: client/src/Component/Pages/Landing/index.js [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+              name="Proceed"
+              onClick={this.handleClick}
+            />
+          </div>
+        ),
+        insert: "top",
+        container: "top-center",
+        dismissable: { click: true }
+      });
+      localStorage.setItem("visit", "true");
```
Comment (IsraaSulaiman) [inline client/src/Component/Pages/Landing/index.js]: I think we should set this in the storage after the user clicks `proceed`. What do you think?

Comment (dubhcait) [reply client/src/Component/Pages/Landing/index.js]: Ya if you load the page and then refresh it currently the pop disappears. I missed that

Comment (engshorouq) [reply client/src/Component/Pages/Landing/index.js]: oh yes, sorry I didn't focus for this point :+1: 
fixed it

**Pull request merged**